### ISSUE TITLE
Client.create() does not resolve when remote debugging is disabled

### ISSUE THREAD

**Issue by jeanverster** [twilio/TwilioChatJsReactNative#55]
I've been unable to get the `client.create` method to resolve while running my app without remote debugging enabled. I assume this is because of differences in JS Core and V8 engines while the remote debugger is active. I'm not sure how to go about debugging what's causing the issue as it doesn't throw an error? It just hangs indefinitely and the memory usage skyrockets. 

I am using Expo SDK 38, with version 4.0.0 of twilio-chat. 

This code doesn't throw an error, just makes the app unresponsive. I have verified the token is being generated correctly, and it throws an error if the token is incorrect. Could it be something to do with Expo? I am at a bit of a loss.

```
React.useEffect(() => {
    async function init() {
      try {
        const client = await TwilioChatClient.create(token)
      } catch (error) {
        console.log('init -> error', error)
      }
    }
    init()
  }, [])
```

**Comment by InfiniteRain** [comment]
It is a very weird behavior indeed. Could you please create a minimal-setup project that recreates this exact issue and send it my way so that I can take a look and debug it?

**Comment by jeanverster** [comment]
@InfiniteRain sure, I will set that up now quick

**Comment by jeanverster** [comment]
Hi @InfiniteRain I was in the process of setting up a recreation and managed to get it working, I realised that console logs were the culprit. I was trying to log out values at the different stages of client / channel initialisation and for some reason that was causing it to hang. With the remote debugger open they worked fine, and as soon as I removed them the problem went away!

Anyways, yeah for anyone coming here because the `create` method isn't resolving while the debugger isn't open - remove any console.logs which are logging out any values which get returned from the `twilio-chat` methods, and that should hopefully fix it. 👍
